### PR TITLE
Efrs 693

### PR DESCRIPTION
**Pull request by pnikalaichyk**

### CODE REVIEW

File: dev/.env [hunk 1]
```diff
@@ -1,4 +1,8 @@
 registry=
 postgres_password=postgres
 postgres_domain=compreface-postgres-db
-postgres_port=5432
\ No newline at end of file
+postgres_port=5432
+email_host=smtp.gmail.com
+email_sender=
+email_password=
+enable_email_server=true
```
Comment (pospielov) [inline dev/.env:8]: Let it be `false` by default

File: admin/src/main/java/com/exadel/frs/helpers/EmailSender.java [hunk 2]
```diff
@@ -28,16 +28,12 @@
...
-    @Value("${spring.mail.sender}")
-    private String frsEmail;
-
     @Autowired
     private JavaMailSender javaMailSender;
 
     public void sendMail(final String to, final String subject, final String message) {
         val msg = new SimpleMailMessage();
         msg.setTo(to);
-        msg.setFrom(frsEmail);
```
Comment (pospielov) [inline admin/src/main/java/com/exadel/frs/helpers/EmailSender.java:40]: Why did you decide to delete it? 
`From` field could contain different value from `username`, also `From` can contain Name of the sender, e.g. '"My Name" <ttt@gmail.com>'

Comment (pnikalaichyk) [reply admin/src/main/java/com/exadel/frs/helpers/EmailSender.java:40]: cos JavaMailSender bean get sender from property (application.yml) spring.mail.username automatically
image

File: dev/.env (hunk 1 above)
Comment (pnikalaichyk) [reply dev/.env]: done

File: admin/src/main/java/com/exadel/frs/helpers/EmailSender.java [hunk 3]
```diff
@@ -17,29 +17,38 @@
...
 @Service
 public class EmailSender {
 
-    @Value("${spring.mail.sender}")
-    private String frsEmail;
+    @Value("${spring.mail.from}")
+    private String from;
+
+    @Value("${spring.mail.username}")
+    private String sender;
```
Comment (pospielov) [inline admin/src/main/java/com/exadel/frs/helpers/EmailSender.java:37]: you defined field but I don't see where you use it.

**Review by pospielov — DISMISSED** [review]

**Review by pospielov — APPROVED** [review]

**Pull request merged**